src/main/java/uk/gov/hmcts/reform/civil/notification/handlers/createsdo/CreateSDOPartiesEmailGenerator.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.reform.civil.notification.handlers.createsdo;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.notification.handlers.EmailDTO;
+import uk.gov.hmcts.reform.civil.notification.handlers.PartiesEmailGenerator;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@AllArgsConstructor
+@Slf4j
+@Component
+public class CreateSDOPartiesEmailGenerator implements PartiesEmailGenerator {
+
+    protected final CreateSDOAppSolOneEmailDTOGenerator appSolOneEmailDTOGenerator;
```

Comment:
It should just be constructed with list of generators and all parties email generator will do the logic